src/SME.SGP.Aplicacao/Interfaces/CasosDeUso/Aula/IExcluirAulaWorkerUseCase.cs

```diff
@@ -0,0 +1,13 @@
+﻿using SME.SGP.Infra;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SME.SGP.Aplicacao.Interfaces.CasosDeUso.Aula
+{
+    public interface IExcluirAulaWorkerUseCase
+    {
+        Task<bool> Executar(MensagemRabbit mensagemRabbit);
+    }
```

Comment:
```suggestion
    public interface IExcluirAulaWorkerUseCase: IUseCase<MensagemRabbit, bool>
    {
    }
```

Criamos essa interface base que já tem um método executar recebendo o TParameter e TResponse do método